tensorflow/python/data/ops/dataset_ops.py

```diff
@@ -611,6 +621,14 @@ def from_tensor_slices(tensors):
            [3, 2]], dtype=int32), array([[b'A'],
            [b'B']], dtype=object))
 
+    >>> # `from_tensor_slices` can also be used to repeat the examples
```

Comment:
I think it's better to remove this section to keep the docstring focused. Each docstring should make sense on its own, but this relies on the user being familiar with the content of the `from_tensors` docstring

Reply:
Done